notebooks/GWAS_blood_pressure_p3/main.py

```diff
@@ -0,0 +1,649 @@
+    * Configure your notebook runtime using the gear wheel at the top right to use a spark cluster for parallel processing. You can learn more about Spark in the Hail section below.
+    * A suggested custom configuration using a 100GB VCF file is below. For larger files, consider using more workers.
+    * Using pre-emptibles is not recommended for analyses that take longer than 6 hours. For large-scale GWAS data preparation, this may be the case depending on the number of workers you request for parallel processing of your variant matrix.
+    <table style="float:left">
+        <thead>
+            <tr><th>Attributes</th><th>Value</th></tr>
+        </thead>
+        <tbody>
+            <tr><td> Application configuration</td><td>Hail: (Python 3.7.9, Spark 2.4.5, hail 0.2.57) </tr></td>
+            <tr><td> CPUs</td><td>8 </tr></td>
+            <tr><td> Memory (GB)</td><td>30 </tr></td>
+            <tr><td> Disk size (GB)</td><td>500 </tr></td>
+            <tr><td> Startup script</td><td>(leave blank) </tr></td>
+            <tr><td> Compute type</td><td>Spark cluster </tr></td>
+            <tr><td> Workers</td><td>120 </tr></td>
+            <tr><td> Preemptible</td><td>50 </tr></td>
+            <tr><td> Workers-CPUs</td><td>8 </tr></td>
+            <tr><td> Workers-Memory (GB)</td><td>30 </tr></td>
+            <tr><td> Workers-Disk size (GB)</td><td>100 </tr></td>
+        </tbody>
+    </table>
+    """
+with herzog.Cell("markdown"):
+    """
+    # Load useful packages
+
+    * **FISS** - a toolkit for using Firecloud/Terra APIs through Python
+    * **Pandas & Numpy** - packages for data analysis
+    * **Pprint** - for pretty printing
+    * **Matplotlib & Seaborn** - for plotting
+    * **Hail** - an open-source, general-purpose, Python-based data analysis tool with additional data types and methods for working with genomic data
+
+    ## Install and update packages
+    """
+with herzog.Cell("python"):
+    #%pip install tenacity
+    #%pip install hail
+    pass
+with herzog.Cell("markdown"):
+    """
+    **Restart the kernel after every pip install. You can do this using the kernel selection in the toolbar.**
+
+    ## Import all the packages this notebook will use
+
+    """
+with herzog.Cell("python"):
+    #%%capture
+    from firecloud import fiss
+    import pandas as pd
+    pd.set_option('display.max_row', 10)
+    import os
+    import io
+    import numpy as np
+    from pprint import pprint
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+    #%matplotlib inline
+    import time
+    from datetime import timedelta
+    import tenacity
+
+fiss = mock.MagicMock()  # noqa # test fixture
+sns = mock.MagicMock()  # noqa test fixture
+pd = mock.MagicMock()  # noqa test fixture
+with herzog.Cell("markdown"):
+    """
+    # Load workspace data
```

Comment:
Remove "load workspace data" and "phenotypic data for each individual...." as its no longer relevant to this notebook.